### PR TITLE
added script to populate spacer + repeat tables

### PR DESCRIPTION
**Pull request by aays**
addresses issue #120

### CODE REVIEW

**Comment by mbonsma** [comment]
I'm pretty sure this works, but I ran into database migration issues when testing it - see #138.  @cemyld, if you're able to confirm that this script works, we can merge it!

File: phageAPI/addspacerandrepeat.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+python addspacerandrepeat.py
+
+'''
+
+
+import os
+import sys
+from Bio import SeqIO
+
+spacerinput = os.listdir('../data/spacers') # these are multiple files
```
Comment (mbonsma) [inline phageAPI/addspacerandrepeat.py]: We don't need to use the `/data/spacers` files - everything is in the `spacerdatabase.txt` file.

File: phageAPI/addspacerandrepeat.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+repeatinput = '../data/DRdatabase.txt'
+
+def populate_spacers():
+	spacerfile = SeqIO.parse('../data/spacerdatabase.txt', 'fasta') 
+	for entry in spacerfile: # that one file in data/
+		spacer = entry.seq
+		spacer, created = Spacer.objects.get_or_create(sequence = spacer)
+	for file in spacerinput: # those files in data/spacers
```
Comment (mbonsma) [inline phageAPI/addspacerandrepeat.py]: So that means you can take out this whole second loop!

**Comment by aays** [comment]
Fixed! The script now only imports data from `DRdatabase.txt` and `spacerdatabase.txt`.

**Comment by mbonsma** [comment]
:clap: great!

**Pull request merged**